vendor/github.com/openshift/api/config/v1/types_infrastructure.go

```diff
@@ -160,6 +160,9 @@ const (
 
 	// EquinixMetalPlatformType represents Equinix Metal infrastructure.
 	EquinixMetalPlatformType PlatformType = "EquinixMetal"
+
+	// AlibabaCloudPlatformType represents Alibaba Cloud infrastructure
+	AlibabaCloudPlatformType PlatformType = "AlibabaCloud"
```

Comment:
Are there corresponding changes to the go.mod and go.sum file for this vendor update?

Reply:
github.com/openshift/api/pull/970 Is the dependent pull request. We are waiting for the branch to open. @gujingit We will need to update the go.mod when the upstream api changes merge.

Reply:
ok. I will update the go.mod when the upstream api changes merge.